dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass AppNotificationDevicesData
+    {
+        // Initializes a new Instance of NotificationDevicesData
+        AppNotificationDevicesData();
+
+        // Checks if Video or Audio Calling is supported
+        static Boolean IsVideoOrAudioCallingSupported();
+
+        // Gets or sets the Camera Device Id
+        String CameraDeviceId{ get; set; };
```

Comment:
What do these values default to? The current system settings?

Consider a method like `static AppNotificationConferencingConfig GetCurrent()` that retrieves Windows' default configuration.

Reply:
Current default here would be empty or null since we are not enumerating or using the devices API here. 
If it's null or empty, then it will be mapped to system default in OS Shell where we are enumerating the devices.

Reply:
defaults for CameraDeviceId/MicrophoneId/SpeakerId would be empty string (L"").
but, 
AppNotificationDevicesData DevicesData{nullptr} ;